### ISSUE TITLE
r3dfox can only be minimized, maximized (moved) and closed through the taskbar

### ISSUE THREAD

**Issue by jonm58** [Eclipse-Community/r3dfox-old#77]
Edit by K4sum1: This is caused by setting the mask width/height to 0 to have the fog cover the whole top bar. It doesn't allow the buttons under it to be clicked. Not sure what a fix would look like here.

### **OS:Windows 11 22000.2538**
![r3dfox privacy mode](https://cdn.discordapp.com/attachments/974909520553853038/1249165363350863902/b7e489ca483f2283.png?ex=66664f66&is=6664fde6&hm=1bac6b09da3437b48d6e4a203f14d2b2048af8a8b9719e3bf6fbe370db8089f6&)
The r3dfox privacy mode icon appears on the minimize button and seems to affect the maximize button and close button as well

[1.mp4](https://github.com/Eclipse-Community/r3dfox/assets/73036321/f686acc0-e5a6-42db-a1df-f0da0cf7785d)

**Comment by the-r3dacted** [comment]
When I was helping you on the Discord, I don't think you need to set the mask-width to anything, and it being 100 is probably the cause here.

Go to about:config and right click and reset widget.ev-native-controls-patch.override-aero-caption-buttons-mask-width

**Comment by jonm58** [comment]
I reset "widget.ev-native-controls-patch.override-aero-caption-buttons-mask-width" no fix
so,I reset "widget.ev-native-controls-patch.override-aero-caption-buttons-mask-height",button look like bad but fix issues problem.
![图片](https://github.com/Eclipse-Community/r3dfox/assets/73036321/f25177c8-4071-4715-9648-0668499c0a99)

**Comment by the-r3dacted** [comment]
Set height back to 1, and play around with width values, maybe there's one that works.

**Comment by jonm58** [comment]
No, only set the height to 0 can be used normally, is not the Windows title bar button and R3dfox title bar conflict?

**Comment by jonm58** [comment]
And this should have something to do with this issues
![图片](https://github.com/Eclipse-Community/r3dfox/assets/73036321/cfa440f8-f1cf-4fba-ad4c-6d2268e5ab59)

**Comment by the-r3dacted** [comment]
I'm not sure if this is even applicable anymore. If it is though, let me know and I'll reopen.
